### PR TITLE
chapter_14

### PR DESCRIPTION
**Pull request by muizzuddinyafiq**
ready

### CODE REVIEW

**Comment by muizzuddinyafiq** [comment]
ready

File: test/fixtures/relationships.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (huongnt-2545) [inline test/fixtures/relationships.yml]: delete this file, it's not useful

File: test/models/relationship_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/models/relationship_test.rb]: delete this file

**Comment by muizzuddinyafiq** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged
You can merge this pull

**Pull request merged**